setup.py

```diff
@@ -25,7 +25,8 @@
 # Dependencies
 extras_require = {
     'serial':   ['pyserial~=3.0'],
-    'neovi':    ['python-ics>=2.12']
+    'neovi':    ['python-ics>=2.12'],
+    'mf4':      ['asammdf>=5.5.0', 'numpy>=1.16.0']
```

Comment:
This would be the coolest, but I don't know whether this actually works:

```suggestion
    'mf4;python_version>="3.6"':      ['asammdf>=5.5.0', 'numpy>=1.16.0']
```

This should work anyways:

```suggestion
    'mf4':      ['asammdf>=5.5.0;python_version>="3.6"', 'numpy>=1.16.0;python_version>="3.6"']
```